src/adapter/src/coord/command_handler.rs

```diff
@@ -137,7 +137,12 @@ impl Coordinator {
             Command::GetSystemVars { session, tx } => {
                 let mut vars = BTreeMap::new();
                 for var in self.catalog().system_config().iter() {
-                    vars.insert(var.name().to_string(), var.value());
+                    if var
```

Comment:
@jkosh44 It seemed like anyone able to access e.g. `GetSystemVars` was able to get all of the system vars without respecting the `internal` flag. I fixed this in the hackiest/least intrusive way possible in all of the call sites, but can you PTAL? I have another more invasive diff that pushes all of these user checks into everything that accesses the system config vars, but it's a sweeping change and figured you probably had some kind of change you envisioned for RBAC so wanted to coordinate.